lib/gooddata/lcm/actions/collect_clients.rb

```diff
@@ -83,10 +85,22 @@ def collect_clients(params, segment_names = nil)
             segment_name = row[segment_id_column]
             GoodData.logger.debug("Segment name: #{segment_name}")
             if segment_names.nil? || segment_names.include?(segment_name)
+              client_id = row[client_id_column]
+              pid = row[project_id_column]
+
+              if pid
+                begin
+                  errors << "Project #{pid} of client #{client_id} is deleted." if client.projects(pid).deleted?
```

Comment:
nope, it's not! This project is not from any clients. This is an abandon project and user want to assign it to a new client in provisioning brick. So this is the right place to check it.